lib/helpers/grenache.service.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const path = require('path')
+process.env.EXPRESS_CONFIG_PATH = path.join(process.cwd(), 'config/grenache.service.conf.json')
```

Comment:
very nice,

maybe we can use `__dirname`, depending on how the service is run and from where process.cwd might be different

Reply:
Oh my god, I have no idea why I put `process.cwd()` in here, of course, it should be `__dirname`, thank you!